src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let countOfCalls = 0;
```

Comment:
```suggestion
  let callsCount = 0;
```

you can make name 2 letters shorter 😅